### ISSUE TITLE
Smart Screen blocks the installer

### ISSUE THREAD

**Issue by CamperSamu** [rauenzi/BetterDiscordApp#113]
![image](https://user-images.githubusercontent.com/32773961/47953664-0310ad80-df81-11e8-9cdb-bf23f09ae8c1.png)
The installer gets blocked by Smart Screen.

**Comment by rauenzi** [comment]
Click "More information" then select "run anyways"

**Comment by CamperSamu** [comment]
![image](https://user-images.githubusercontent.com/32773961/47953828-fee58f80-df82-11e8-95f6-63d7611d1043.png)
AV blocks it too

**Comment by rauenzi** [comment]
So exclude it?

**Comment by CamperSamu** [comment]
I've already done this, ik how to bypass smartscreen, it was just to report
the fact 09

Il sabato 3 novembre 2018, Zack <notifications@github.com> ha scritto:

**Comment by rauenzi** [comment]
I already know this and has been reported before. I can't get around it without paying to have the executable signed, and as this is an open source project I can't really afford that.
